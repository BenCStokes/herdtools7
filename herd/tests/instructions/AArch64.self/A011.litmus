AArch64 A011
{
0:X2=0;
0:X0=NOP;
0:X3=P0:L1;
}
      P0       ;
  STR W0,[X3]  ;
L1:            ;
  B L0         ;
  ADD W2,W2,#1 ;
L0:            ;
exists (0:X2=0)

