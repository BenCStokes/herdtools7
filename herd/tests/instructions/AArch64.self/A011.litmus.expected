Test A011 Allowed
States 2
0:X2=0;
0:X2=1;
Ok
Witnesses
Positive: 1 Negative: 1
Condition exists (0:X2=0)
Observation A011 Sometimes 1 1
Hash=7c2889d6cfa4501cd73ee70ba35c713b

